app/src/androidTest/java/de/grobox/transportr/networks/PickTransportNetworkActivityTest.kt

```diff
@@ -74,13 +74,13 @@ class PickTransportNetworkActivityTest : ScreengrabTest() {
         makeScreenshot("1_FirstStart")
 
         // hack to find region position in list
-        val regionList = ArrayList(EnumSet.allOf(Region::class.java))
+        val regionList = ArrayList(EnumSet.allOf(Country::class.java)) //TODO: pretty sure it will break :/
```

Comment:
If you are pretty sure this will break, is this PR then even ready for review or did you just forget to remove this TODO?

Reply:
I'll look into Android tests, I've never worked with them before. But if you could give me a hint on how this test was written, would be great!

Reply:
This is an instrumentation test running on a real device, simulating user input. It is primarily used to take nice screenshots automatically. It is *not* run by the CI.

So you need to attach an Android device via ADB and run `All tests` in Android Studio.

Reply:
Tests are now running, thanks to the continent ordering even simplier :)